pkg/kapp/cmd/app/deploy.go

```diff
@@ -264,6 +264,15 @@ func (o *DeployOptions) newAndUsedGKs(newGKs []schema.GroupKind, app ctlapp.App)
 		}
 	}
 
+	podGK := schema.GroupKind{
+		Kind: "Pod",
+	}
+
+	// adding Pod in GKs to get existing Pod resources (#460)
+	if _, exists := gksByGK[podGK]; !exists && o.DeployFlags.Logs {
+		uniqGKs = append(uniqGKs, podGK)
```

Comment:
I think we can do this inline 
```suggestion
		uniqGKs = append(uniqGKs, schema.GroupKind{Kind: "Pod"})
```
A variable seems a bit unnecessary

Reply:
But we are using `podGk` in this statement also before appending here - 
> if _, exists := gksByGK[podGK]; !exists && o.DeployFlags.Logs {

Reply:
Oh I see, my bad, LGTM then

Reply:
That's why I thought have the podGk as variable and using it in both the places